[DOC]Update custom image guide in GCP dataproc to reduce cluster startup time (#2666)
NVIDIA/spark-rapids: docs/get-started/getting-started-gcp.md

* [GPU Driver](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/gpu) and
  [RAPIDS Acclerator for Apache
  Spark](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids) through

>   initialization actions (please note it takes up to 1 week for latest commit to be merged into gcp
>   Dataproc public GCS bucket)

Reviewer: ```suggestion
  initialization actions (please note it takes up to 1 week for the latest init script to be merged into the GCP
  Dataproc public GCS bucket)
```

application](https://github.com/NVIDIA/spark-xgboost-examples/tree/spark-3/examples/apps/scala).
After [building the jar
files](https://github.com/NVIDIA/spark-xgboost-examples/blob/spark-3/getting-started-guides/building-sample-apps/scala.md)

> they are available through maven `mvn package -Dcuda.classifier=cuda11-0`. In 21.06 release, cuda 11.0/11.2 will be supported. 

Reviewer: Request to keep lines to <= 100 characters for readability.
```suggestion
they are available through maven `mvn package -Dcuda.classifier=cuda11-0`. In the 21.06 release, 
CUDA 11.0/11.2 will be supported. 
```

can use example [pyspark notebooks](../demo/GCP/Mortgage-ETL-GPU.ipynb) to experiment.

## Build custom dataproc image to accelerate cluster init time
In order to accelerate cluster init time to 3-4 minutes, we need to build a custom Dataproc image
that already has NVIDIA drivers and CUDA toolkit installed, with RAPIDS deployed. The custom image 
could also be used in an air gap environment. In this section, we will be using [these
instructions from GCP](https://cloud.google.com/dataproc/docs/guides/dataproc-images) to create a
custom image. 

> Currently, the [GPU Driver](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/gpu) initialization actions:

Reviewer: Request to keep lines to <= 100 characters for readability.
```suggestion
Currently, the [GPU Driver](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/gpu) 
initialization actions:
```

export ZONE=[Your Preferred GCP Zone]
export GCS_BUCKET=[Your GCS Bucket]
export IMAGE_NAME=a209-ubuntu18-gpu-t4
export DATAPROC_VERSION=2.0.9-ubuntu18
export GPU_NAME=nvidia-tesla-t4
export GPU_COUNT=1

python generate_custom_image.py \
    --image-name $IMAGE_NAME \
    --dataproc-version $DATAPROC_VERSION \
    --customization-script $CUSTOMIZATION_SCRIPT \
    --no-smoke-test \
    --zone $ZONE \
    --gcs-bucket $GCS_BUCKET \
    --machine-type n1-standard-4 \
    --accelerator type=$GPU_NAME,count=$GPU_COUNT \
    --disk-size 200
```

See [here](https://cloud.google.com/dataproc/docs/guides/dataproc-images#running_the_code) for more
details on `generate_custom_image.py` script arguments and 
[here](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions) for dataproc version description.

The image `sample-209-ubuntu18-gpu-t4` is now ready and can be viewed in the GCP console under
`Compute Engine > Storage > Images`. The next step is to launch the cluster using this new image and
new initialization actions (that do not install NVIDIA drivers since we are already past that step).

> Here is the new custom GPU initialization action that needs to be run after cluster creation with custom images, save this as `addon.sh`

Reviewer: Request to keep lines to <= 100 characters for readability.  
```suggestion
Here is the new custom GPU initialization action that needs to be run after cluster creation with 
custom images, save this as `addon.sh`
```
